Modules/Core/Common/wrapping/itkImageToImageFilterB.wrap

```diff
@@ -32,13 +32,18 @@ itk_wrap_class("itk::ImageToImageFilter" POINTER)
   itk_wrap_image_filter_combinations("${WRAP_ITK_COMPLEX_REAL}" "${WRAP_ITK_SCALAR}")
 
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
-    foreach(t ${WRAP_ITK_SCALAR})
-      # scalar <-> SymmetricSecondRankTensor
-      itk_wrap_template("${ITKM_I${t}${d}}${ITKM_ISSRT${ITKM_D}${d}${d}}" "${ITKT_I${t}${d}}, ${ITKT_ISSRT${ITKM_D}${d}${d}}")
-      itk_wrap_template("${ITKM_ISSRT${ITKM_D}${d}${d}}${ITKM_I${t}${d}}" "${ITKT_ISSRT${ITKM_D}${d}${d}}, ${ITKT_I${t}${d}}")
+    foreach(t2 ${WRAP_ITK_REAL})
```

Comment:
We still need to always wrap for `double`, so, e.g.:

```
UNIQUE(real_types "${WRAP_ITK_REAL};D")

[...]
foreach(t2 ${real_types})
```

Reply:
Fixed!